src/components/dialog/footer/ManagerProgressFooter.vue

```diff
@@ -96,14 +98,22 @@ const currentTaskName = computed(() => {
 })
 
 const handleRestart = async () => {
-  await useComfyManagerService().rebootComfyUI()
-  closeDialog()
-
-  const onReconnect = () => {
-    useCommandStore().execute('Comfy.RefreshNodeDefinitions')
+  const onReconnect = async () => {
+    // Refresh manager state
     comfyManagerStore.clearLogs()
     comfyManagerStore.setStale()
+
+    // Refresh node definitions
+    useCommandStore().execute('Comfy.RefreshNodeDefinitions')
```

Comment:
Why is this async command is not awaited?
https://github.com/Comfy-Org/ComfyUI_frontend/blob/8f00d8ca6a5da965d1a86f73d1b70c9744c263d1/src/composables/useCoreCommands.ts#L171-L178
https://github.com/Comfy-Org/ComfyUI_frontend/blob/8f00d8ca6a5da965d1a86f73d1b70c9744c263d1/src/stores/commandStore.ts#L89-L99